webviz_subsurface/plugins/_parameter_analysis/models/parameters_model.py

```diff
@@ -206,39 +206,94 @@ def make_grouped_plot(
                         ),
                         20,
                     ),
+                    facet_row_spacing=max((0.08 - (0.00071 * len(parameters))), 0.03),
                     color="ENSEMBLE",
                     color_discrete_sequence=self.colorway,
+                    custom_data=["PARAMETER"],
                 )
                 .update_xaxes(
                     matches=None,
                     fixedrange=True,
                     title=None,
-                    showticklabels=(len(parameters) < 20),
+                    showticklabels=len(parameters) <= 100,
+                    tickangle=0,
+                    tickfont_size=max((18 - (0.4 * len(parameters))), 8),
```

Comment:
Similar thing here, not sure how it works on different screen sizes and resolutions?

Reply:
Works fine on laptop and big monitor 👍